components/OverviewSection.vue

```diff
@@ -1,38 +1,18 @@
 <template>
   <section id="overview">
     <v-container>
-      <v-stack
-        align="start"
-        direction="column"
-        sm-direction="column"
-        gap="1.875rem"
-        sm-gap="1.25rem"
-      >
-        <v-heading
-          class="overview-heading"
-          :level="2"
-          line-height="1.2"
-          :weight="700"
-        >
+      <v-stack align="start" direction="column" sm-direction="column" gap="1.875rem" sm-gap="1.25rem">
+        <v-heading class="overview-heading" :level="2" line-height="1.2" :weight="700">
           Overview
         </v-heading>
         <div>
-          <v-text
-            class="overview-description"
-            color="secondary"
-            :weight="400"
-            line-height="1.4"
-          >
+          <v-text class="overview-description" color="secondary" :weight="400" line-height="1.4">
```

Comment:
line-height is very specific, can we change it to `1.5`

Reply:
Resolved at: https://github.com/arcana-network/testnet-website/pull/23/commits/28faa2a5f380dd3b85649a91b750d7b3bf0b18f1